.github/workflows/coverage.yml

```diff
@@ -0,0 +1,52 @@
+# SPDX-FileCopyrightText: © Jens Bergmann and contributors
+# SPDX-License-Identifier: AGPL-3.0-or-later
+# SPDX-ArtifactOfProjectHomePage: https://github.com/Second-Hand-Friends/kleinanzeigen-bot
+#
+# .github/workflows/coverage.yml
+name: Post coverage comment
```

Comment:
does this need to be a separate workflow?

Reply:
We are not relying on pull_request and we set token permissions on job level so I don't think this setup is actually required.